<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QMainWindow" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>516</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Session</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="lblIntro">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>471</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Welcome to RemoteCtl! This program allows you to connect to your server easily and efficiently, and lets you open SSH terminals, explore files, and edit them all in one place.</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lblHeader">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;progname-version&gt;</string>
    </property>
   </widget>
   <widget class="QFrame" name="frmNew">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>471</width>
      <height>151</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>54</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Host:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>User Name:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>80</y>
       <width>61</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>TCP Port:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Login Directory:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtHost">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>401</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtUser">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>80</y>
       <width>211</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtPort">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>80</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>22</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtDir">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>110</y>
       <width>341</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>/</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Session Identifier:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtId">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>331</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QRadioButton" name="radioNew">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>96</width>
      <height>23</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>&amp;New Session</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QFrame" name="frmOpen">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>471</width>
      <height>121</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QListWidget" name="lstOpen">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>451</width>
       <height>91</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Item1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Item2</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QRadioButton" name="radioOpen">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>320</y>
      <width>96</width>
      <height>23</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>&amp;Open Session</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnQuit">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>470</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Quit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLogin">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>470</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Login</string>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDelete">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>470</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Delete</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblIcon">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/icons/remotectl.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
